pom.xml

```diff
@@ -319,12 +319,12 @@ under the License.
       <dependency>
         <groupId>org.apache.maven.shared</groupId>
         <artifactId>maven-shared-utils</artifactId>
-        <version>3.2.1</version>
+        <version>3.3.4-SNAPSHOT</version>
```

Comment:
We can't do this. When bisecting down to this commit it will be virtually impossible to reproduce. I went through with Maven Artifact Transfer.

Reply:
The only relevant commit is https://github.com/apache/maven/pull/436/commits/690aff3ffb08fc5366f72736f668775e86ba67e9.  The two ones before are part of https://github.com/apache/maven/pull/436/commits/690aff3ffb08fc5366f72736f668775e86ba67e9 which itself depends on https://github.com/apache/maven-shared-utils/pull/69.
Things have to be done step by step...